packages/eas-json/src/build/types.ts

```diff
@@ -52,6 +53,12 @@ export interface IosBuildProfile extends CommonBuildProfile {
   artifactPath?: string;
   scheme?: string;
   buildConfiguration?: string;
+
+  ascApiKeyPath?: string;
+  ascApiKeyIssuerId?: string;
+  ascApiKeyId?: string;
+  appleTeamId?: string;
+  appleTeamType?: AppleTeamType;
```

Comment:
I would prefer not to configure that via eas.json, but I don't have strong opinion on that. If you decide to stay with this approach wrap all those fields in some object, so we are clear that e.g appleTeamId is required for asc API key not for other build stuff